### ISSUE TITLE
Adding aggregation to Search using A

### ISSUE THREAD

**Issue by benregn** [elastic/elasticsearch-dsl-py#168]
Hey, how would one go about adding an aggregation to a Search when the aggregation was defined using the A class?  It's not covered in the [aggregation docs](http://elasticsearch-dsl.readthedocs.org/en/latest/search_dsl.html#aggregations).

**Comment by benregn** [comment]
Figured it out :)

```
a = A('terms', field='category')
s = Search()
s.aggs.metric('category_terms', a)
```

**Comment by HonzaKral** [comment]
Fixed by #169 
Thanks!
